src/observable/BoundCallbackObservable.ts

```diff
@@ -15,6 +15,7 @@ export class BoundCallbackObservable<T> extends Observable<T> {
   subject: AsyncSubject<T>;
 
   /* tslint:disable:max-line-length */
+  static create(callbackFunc: (callback: () => any) => any, selector?: void, scheduler?: IScheduler): () => Observable<void>;
```

Comment:
Now that I think about it, I have seen some operators have type-only tests. Maybe I should add ones here?